src/types_10720.ts

```diff
@@ -0,0 +1,77 @@
+import type {
+  RegistryTypes,
+} from "@polkadot/types/types"
+
+export const types10720: RegistryTypes = {
```

Comment:
It's sufficient to just define DID related types here right?

Reply:
Yes, since all the runtime types are taken from the metadata. We only need to declare types that are used for RPC endpoints.